src/js/components/ServiceDetailConfigurationTab.js

```diff
@@ -1,19 +1,100 @@
+import {Dropdown} from 'reactjs-components';
 import React from 'react';
 
 import ConfigurationView from './ConfigurationView';
+import DCOSStore from '../stores/DCOSStore';
 import Service from '../structs/Service';
 
+const METHODS_TO_BIND = [
+  'handleVersionSelection'
+];
+
 class ServiceDetailConfigurationTab extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      selectedVersionID: null
+    };
+
+    METHODS_TO_BIND.forEach(method => {
+      this[method] = this[method].bind(this);
+    });
+  }
+
+  componentWillMount() {
+    this.setState({
```

Comment:
After I've added this I removed it again, because it makes the tests fail on the server for clear reasons.
componentWillMount is part of ReactDOM or React Native, which is not existent in this test and _not_ part of React.Component itself.
It's also a little bit doubtfully to me to call this explicitly without any actual need. That is against the principle of using things only if you need them.